### PR TITLE
Feature/landing page

### PR DESCRIPTION
**Pull request by itsninam**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,21 +1,24 @@
 // Modules
-import { useState, useEffect } from "react";
+import { useState } from "react";
+import { Route, Routes } from "react-router-dom";
+
 // Components
 import Header from "./Components/Header.js";
 import Footer from "./Components/Footer.js";
 import LeaderboardForm from "./Components/LeaderboardForm.js";
 import FetchData from "./Components/FetchData.js";
```
Comment (alextfriedman) [inline src/App.js:9]: we changed the name of this component, so we'll need to update this

**Review by benbourgon — APPROVED** [review]

**Review by mwazir — APPROVED** [review]

File: src/styles/sass/base/_baseStyles.scss [hunk 2]
```diff
@@ -1,29 +1,54 @@
...
 ul {
-    list-style: none;
-    @include resetBox;
+  list-style: none;
+  @include resetBox;
 }
 
 a {
+  color: $clr-limeGreen;
+  &:visited {
```
Comment (alextfriedman) [inline src/styles/sass/base/_baseStyles.scss:26]: need to make sure the colour changes back when you toggle away

**Review by alextfriedman — CHANGES_REQUESTED** [review]
(fort) Awesome job 🦝

**Comment by itsninam** [comment]
merging after PR review

**Pull request merged**